feature-staking-impl/src/main/java/io/novafoundation/nova/feature_staking_impl/presentation/mappers/Pool.kt

```diff
@@ -11,22 +11,23 @@ import io.novafoundation.nova.feature_staking_impl.domain.nominationPools.model.
 import io.novafoundation.nova.feature_staking_impl.domain.nominationPools.model.apy
 import io.novafoundation.nova.feature_staking_impl.presentation.nominationPools.common.PoolDisplayFormatter
 import io.novafoundation.nova.feature_staking_impl.presentation.pools.common.PoolRvItem
+import io.novafoundation.nova.runtime.ext.addressOf
 import io.novafoundation.nova.runtime.multiNetwork.chain.model.Chain
 
-private const val ICON_SIZE_DP = 24
-
 suspend fun mapNominationPoolToPoolRvItem(
     chain: Chain,
     pool: NominationPool,
     resourceManager: ResourceManager,
     poolDisplayFormatter: PoolDisplayFormatter,
     isChecked: Boolean
 ): PoolRvItem {
+    val model = poolDisplayFormatter.format(pool, chain)
     return PoolRvItem(
         id = pool.id.value,
-        model = poolDisplayFormatter.format(pool, chain),
+        model = model,
         subtitle = getSubtitle(pool, resourceManager),
         members = pool.membersCount.format(),
+        address = chain.addressOf(model.poolAccountId),
```

Comment:
We should make `address` a part of `PoolDisplayModel` instead since we already calculate address in `poolDisplayFormatter`